docs/getting-started-monitoring.md

```diff
@@ -1,6 +1,6 @@
 # Getting Started with Helm Charts (Monitoring using Prometheus Operator)
 
-This document explains how to get started with Scalar products monitoring on Kubernetes using Prometheus Operator (kube-prometheus-stack). Here, we assume that you already have a Mac or Linux environment for testing.  
+This document explains how to get started with Scalar products monitoring on Kubernetes using Prometheus Operator (kube-prometheus-stack) and Loki Stack (loki + promtail). Here, we assume that you already have a Mac or Linux environment for testing.
```

Comment:
I think we should separate the doc into two parts: Monitoring using Prometheus operator and Log aggregation using Locki.
The latter depends on Grafana so we can make the latter depend on the former doc by saying something like `Please install Prometheus operator ...`.

What do you think? > @supl @ymorimo

Reply:
Understood.
Let me create another getting-started for it.

Reply:
I create another getting-started document in https://github.com/scalar-labs/helm-charts/pull/87/commits/d32769fe84b9f9e0bf1da12e31d1bc4dd43eb825